generator/toyota/_toyota_nodsu_common.dbc

```diff
@@ -11,6 +11,11 @@ BO_ 401 STEERING_LTA: 8 XXX
  SG_ STEER_REQUEST_2 : 0|1@0+ (1,0) [0|1] "" XXX
  SG_ SETME_X1 : 7|1@0+ (1,0) [0|1] "" XXX
 
+BO_ 956 GEAR_PACKET: 8 XXX
```

Comment:
This looks to be generic across all supported models, we can add this to `_toyota_2017.dbc`, there are also some extra signals
```
BO_ 956 GEAR_PACKET: 8 XXX
 SG_ SPORT_ON : 2|1@0+ (1,0) [0|1] "" XXX
 SG_ GEAR : 13|6@0+ (1,0) [0|63] "" XXX
 SG_ SPORT_GEAR : 38|3@0+ (1,0) [0|7] "" XXX
 SG_ ECON_ON : 40|1@0+ (1,0) [0|1] "" XXX
 SG_ B_GEAR_ENGAGED : 41|1@0+ (1,0) [0|1] "" XXX
 SG_ DRIVE_ENGAGED : 47|1@0+ (1,0) [0|1] "" XXX
```

Reply:
I suggest we start with nodsu_hybrid since it'll be easier to verify.

Reply:
I think ` SG_ GEAR : 13|6@0+ (1,0) [0|63] "" XXX` is already in most generator files except nodsu_hybrid and Prius, I've already verified that this works on the Prius, so we just need to test nodsu_hybrid

Reply:
the first step is to get as many affected cars tested as possible. can you and @sshane reach out to users on discord with the affected cars and have them cycle through each gear?

Reply:
> then, it should be easy to write a quick script/notebook to verify it matches the output of the old signal

We can ask the discord users to cycle their gears in the order that @sshane did, `P -> R -> N -> D -> N -> R -> P -> R -> D`, so it'll be easy to compare and verify.

Reply:
hmm, I wouldn't rely on the order. just check that the old and new signal matches and that each gear is seen